src/cli/options.rs

```diff
@@ -2189,28 +2154,20 @@ impl Default for HistoryOptions {
     }
 }
 
-impl fmt::Display for HistoryOptions {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let mut s = if self.short { "short" } else { "full" }.to_string();
-
+impl HistoryOptions {
+    pub fn url_path_parameters(&self) -> String {
         if let Some(before) = self.before {
             let after = self.after.map(|t| t.timestamp()).unwrap_or_else(|| 0);
-            s.push_str(&format!(
-                "/{}/{}/{}/{}",
-                self.rows,
-                self.offset,
-                after,
-                before.timestamp()
-            ));
+            format!("{}/{}/{}/{}", self.rows, self.offset, after, before.timestamp())
         } else if let Some(after) = self.after {
-            s.push_str(&format!("/{}/{}/{}", self.rows, self.offset, after.timestamp()));
+            format!("{}/{}/{}", self.rows, self.offset, after.timestamp())
```

Comment:
Is it correct to drop the leading forward slash in these else blocks?

Reply:
Yeah, there were more changes here. It's no longer appending this to `s`